Makefile

```diff
@@ -43,14 +48,15 @@ push:
 
 check: fmt vet
 
-test:
+test: libvirt
```

Comment:
You are setting CONTAINERIZED=1 here https://github.com/openshift/release/pull/1415/files#diff-8494a5bc314f945b17df41d8bb54cfc9R32
So CGO_deps won't execute so woulnd't this still fail?

Reply:
You are right. I fixed condition so it should work now.